ui/app/templates/components/selectable-card-container.hbs

```diff
@@ -0,0 +1,74 @@
+{{#linked-block
+  "vault.cluster.metrics.http-requests"
+  class="grid-item-http"
+}}
+  <SelectableCard 
+    @cardTitle="Http Requests" 
+    @total={{totalHttpRequests}}
+    @subText="Total"
+    @gridContainer={{gridContainer}}
+  >
+  {{#if (eq counters.httpsRequests.length 1)}}
+    <div class="is-paddingless is-marginless">
+      <div class="level-item">
+        <PopupMenu name="engine-menu">
+          <nav class="menu">
+            <ul class="menu-list">
+              <li class="action">
+                {{#link-to "vault.cluster.metrics.http-requests"}}
+                  View Details
+                {{/link-to}}
+              </li>
+            </ul>
+          </nav>
+        </PopupMenu>
+      </div>
+    </div>
+  {{else}}
+    <div class="change-metric">
+      <Icon @glyph="arrow-up" @size="xlm" aria-hidden="true" class="change-metric-icon {{if (gt percentChange 0) "is-increase" "is-decrease"}}"/>
+      {{#if (gt percentChange 0)}}
+        <p class="has-text-grey is-size-4 amount-change">+{{percentChange}}%</p>
+      {{else}}
+        <p class="has-text-grey is-size-4 amount-change">{{percentChange}}%</p>
```

Comment:
does this need a `-` to show it's x% less than the previous month?

Reply:
The number comes in automatically as a negative, with the `-` prefixed, so it doesn't.  But good thought.  I had to double-check myself to make sure.  Here's a screenshot. 
![image](https://user-images.githubusercontent.com/6618863/74365592-a1e2a700-4d8b-11ea-8cd7-badd4fbcb92b.png)